File: ExercicioCozinha/Ingrediente.java
package SquadraNewThinkers_1T2021.ExercicioCozinha;

//import java.util.Date;

public class Ingrediente {
   private String nome;
   private String validade;

   public Ingrediente(String nome, String validade){
       this.nome = nome;
       this.validade = validade;
   }

   public String getNome(){
       return nome;
   }

   public String getValidade(){
       return validade;
   }

    public String toString() {
        return "\nNome: " + getNome() + "\nData de Validade: " + getValidade();
    }
}

File: PHP_atividade1/classes/Ingrediente.php
<?php

class Ingrediente
{
    private String $nome;
    private DateTime $validade;

    public function __construct(string $nome, DateTime $validade)
    {
        $this->nome = $nome;
        $this->validade = $validade;
    }

    public function getNome(): string
    {
        return $this->nome;
    }

    public function setNome(string $nome): void
    {
        $this->nome = $nome;
    }

    public function getValidade(): DateTime
    {
        return $this->validade;
    }

    public function setValidade($validade): void
    {
        $this->validade = $validade;
    }
}